### PR TITLE
Fix missing translation when uploading an invalid MNO requests file

### PR DESCRIPTION
**Pull request by jamesrose**
### Context

We were missing a translation for an error message when a MNO uploads an invalid CSV file. This PR adds the string and a test case.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -425,6 +425,10 @@ en:
               theres_a_problem_with_that_spreadsheet: There’s a problem with that spreadsheet
               blank: Select a spreadsheet file
               unsupported_file_type: Choose an ‘xlsx’ (Open XML format) Excel file
+        mno/csv_status_update_form:
+          attributes:
+            upload:
+              unsupported_file_type: Choose an CSV file
```
Comment (benilovj) [inline config/locales/en.yml:431]:
```suggestion
              unsupported_file_type: Choose a CSV file
```

File: spec/features/mno/extra_mobile_data_requests_csv_update_spec.rb [hunk 2]
```diff
@@ -85,6 +95,16 @@ def and_i_select_my_large_csv_file_with_errors
     attach_file('CSV file', filename)
   end
 
+  def and_i_select_an_invalid_file
+    File.write(invalid_filename, 'test')
+    attach_file('CSV file', invalid_filename)
+  end
+
+  def then_i_see_an_invalid_error
+    expect(page).to have_text('Choose an CSV file')
```
Comment (benilovj) [inline spec/features/mno/extra_mobile_data_requests_csv_update_spec.rb:104]:
```suggestion
    expect(page).to have_text('Choose a CSV file')
```

**Review by benilovj — APPROVED** [review]
LGTM bar typos

File: config/locales/en.yml (hunk 1 above)
Comment (jamesrose) [reply config/locales/en.yml]: @benilovj Great, thanks!

**Pull request merged**